src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/Example1.java

```diff
@@ -10,18 +10,43 @@
 // xdoc section -- start
 // violation 5 lines below '<p> tag should be preceded with an empty line'
 /**
- * No tag (ok).
+ * No tag
  *
- * <p>Tag immediately before the text (ok).
- * <p>No blank line before the tag (violation).
+ * <p>Tag immediately before the text
+ * <p>No blank line before the tag
  *
  * <p>
- * New line after tag (violation).
+ * New line after tag
  *
- * <p> Whitespace after tag (violation).
+ * <p> Whitespace after tag
  *
+ * <p><b>p tag before inline tag B, this is ok</b></p>
  */
-// violation 3 lines above 'tag should be placed immediately before the first word'
+// violation 4 lines above 'tag should be placed immediately before the first word'
 public class Example1 {
+  // violation 4 lines below '<p> tag should not precede HTML block-tag '<pre>''
```

Comment:
Please make Examples to be same by size, Linux numbers.
Please position  classes and methods to same line numbers.
You can archive this by placing empty line before class

Reply:
Put few empty lines after class to keep foo1 on same line number in both examples.

Reply:
Put 2 empty line after foo1 to keep foo2 to be on same line numbers

Reply:
Sure, doing it